new/asset-catalog/egeria-asset-details.component.js

```diff
@@ -73,6 +73,10 @@ class EgeriaAssetDetails extends mixinBehaviors([EgeriaItemUtilsBehavior], Polym
                             title="Type"
                             with-row-stripes ></egeria-props-table>
 
+        <egeria-props-table items="[[ _attributes(item.origin) ]]"
```

Comment:
is this optional? or a mandatory property which is always there

Reply:
it's a mandatory property